sidecar/src/log.rs

```diff
@@ -243,7 +243,7 @@ impl<S: Subscriber> Filter<S> for &MultiEnvFilter {
             .unwrap()
             .values()
             .map(|f| f.max_level_hint())
-            .min()
+            .max()
```

Comment:
The whole _ordering_ implementations for `Level` and `LevelFilter` makes my head spin since the actual underlying `enum` for both is ordered the other way around, i.e. `Debug` is `1` and `Warn` is `3`, but apparently `Debug` is greater than `Warn`.

Reply:
Yes, that's exactly why I made the mistake in the first place. Really a what the fuck moment :-D